### ISSUE TITLE
Typo in Wiki

### ISSUE THREAD

**Issue by edwardmarks** [ParsePlatform/parse-server#596]
In [Using Parse SDKs with Parse Server](https://github.com/ParsePlatform/parse-server/wiki/Parse-Server-Guide#using-parse-sdks-with-parse-server)

Swift code uses `localhost.com` instead of `localhost`.

Code was:
```
let configuration = ParseClientConfiguration {
    $0.applicationId = "YOUR_APP_ID"
    $0.clientKey = ""
    $0.server = "http://localhost.com:1337/parse"
}
Parse.initializeWithConfiguration(configuration)
```

Should be:
```
let configuration = ParseClientConfiguration {
    $0.applicationId = "YOUR_APP_ID"
    $0.clientKey = ""
    $0.server = "http://localhost:1337/parse"
}
Parse.initializeWithConfiguration(configuration)
```

**Comment by drew-gross** [comment]
Thanks! I've fixed it.

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.3.0-alpha.27](https://github.com/parse-community/parse-server/releases/tag/5.3.0-alpha.27)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.4.0-beta.1](https://github.com/parse-community/parse-server/releases/tag/5.4.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.4.0-alpha.1](https://github.com/parse-community/parse-server/releases/tag/5.4.0-alpha.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.4.0](https://github.com/parse-community/parse-server/releases/tag/5.4.0)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.4.0](https://github.com/parse-community/parse-server/releases/tag/5.4.0)
